autolink/postlink/path.js

```diff
@@ -6,8 +6,10 @@ var ignoreFolders = {
 exports.mainActivityJava = glob.sync('**/MainActivity.java', ignoreFolders)[0];
 exports.mainActivityKotlin = glob.sync('**/MainActivity.kt', ignoreFolders)[0];
 var mainApplicationJava = glob.sync('**/MainApplication.java', ignoreFolders)[0];
+var mainApplicationKotlin = glob.sync('**/MainApplication.kt', ignoreFolders)[0];
+var mainApplication = mainApplicationJava || mainApplicationKotlin
 exports.mainApplicationJava = mainApplicationJava;
```

Comment:
I left mainApplicationJava, as i'm not sure if we do want to change this variable. please change it as well if it's needed